CONTRIBUTING.md

```diff
@@ -182,7 +182,7 @@ test("Factorial", () => {
 
 ```ts
 // More concise, plus we get test name formatting
-test("Factorial", () => {
+describe("Factorial", () => {
```

Comment:
Should we update line 173 as well?

Reply:
No, line 173 uses `test` appropriately (a single test with multiple expects).